TreeNode.cpp

```diff
@@ -10,19 +10,19 @@ void TreeNode::setValue(int value) {
     _value = value;
 }
 
-TreeNode* TreeNode::getLeftChild() const {
+TreeNode * TreeNode::getLeftChild() const {
```

Comment:
Ну и зачем так делать...
```suggestion
TreeNode* TreeNode::getLeftChild() const {
```
"'*" относится к типу возвращаемого значения, поэтому разумно писать именно так.